lumicks/pylake/kymotracker/kymotracker.py

```diff
@@ -251,6 +285,7 @@ def track_lines(
     continuation_threshold=0.005,
     angle_weight=10.0,
     rect=None,
+    refine=True,
```

Comment:
Considering that skipping this step leads to bad results, and we already automatically do this refinement for `track_greedy` I don't think we need to have this option. It keeps things simpler and more consistent I think.

 We can always add it back if needed in the future, specifically if we start using this algorithm for 2D tracking or something like that.

Reply:
I think you need fewer points around the line when you don't do refinement, which could be a benefit. But I suppose we don't really need that right now. Amended!

Reply:
One thing to note is that one test was using this algorithm without refinement. That test seems to find some spurious single point peaks now (that could easily be filtered with `filter_lines`). We could fix this behaviour in a future PR. I can also make an extra commit to add a `min_length` parameter to this one.

Reply:
we end up with single point lines from `track_greedy` so to me it's not a problem. I had suggested adding a `min_length` arg to `track_greedy` but we decided against it, I forget why though. But here, it's just consistent behaviour

Reply:
Right. I'll leave it as is.